Lib/test/test_tarfile.py

```diff
@@ -3056,6 +3057,94 @@ def test_keyword_only(self, mock_geteuid):
                               tarfl.extract, filename_1, TEMPDIR, False, True)
 
 
+_can_chmod_set_sticky = None
+
+
+def can_chmod_set_sticky():
+    """Can chmod set the sticky bit on a file?"""
+
+    def can_chmod_set_sticky_inner():
+        if not os_helper.can_chmod():
+            return False
+        filename = os_helper.TESTFN + "-chmod-sticky"
```

Comment:
honestly, just use "chmod-sticky", to avoid further troubles with non-ASCII filenames.